charts/incubator/rsshub/values.yaml

```
@@ -0,0 +1,25 @@
+image:
+  repository: diygod/rsshub
+  pullPolicy: IfNotPresent
+  tag: 2022-03-19@sha256:53f7237edf291c7f2b3343161245ed658b43faff07ac260dec280f42681003fe
+
+browserlessimage:
+  repository: browserless/chrome
+  tag: latest@sha256:751dedb5701648844b2b91077cf4bb027efc1a4e0bec053f8d8a6b81802dfe86
+
+env:
+  NODE_ENV: production
+  CACHE_TYPE: memory
+  PUPPETEER_WS_ENDPOINT: 'ws://browserless:3000'
```

```suggestion
  PUPPETEER_WS_ENDPOINT: 'ws://localhost:3000'
```
i'm not sure container names work here,, localhost will